snap/snapcraft.yaml

```diff
@@ -31,21 +31,20 @@ parts:
       - pkg-config
       - libsqlite3-dev
       - gettext
-      - libclang-8-dev
-      - on arm64,armhf,ppc64el,s390x:
-        - lld-8
+      - libclang-9-dev
+      - lld-9
```

Comment:
maybe we should get rid of these dependencies again, so we can have broader platform support

Reply:
These actuality exist everywhere; I just dropped the (attempted) “*only* install lld on tier 2 platforms” logic.

With lld-9 the build failures on ppc64 and armhf are elsewhere.